imagersite/imager_images/templates/library.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+{% block content %}
+
+<h1>{{ user.username }}'s Albums</h1>
+<div class="container">
+     {% for album in user.albums.all %}
+        <div class="col-lg-3 col-md-4 col-sm-5 col-xs-6">
+        <h4 class='text-center'>{{ album.title }}</h4>
+        <a href='/images/albums/{{ album.id }}'><img style='box-shadow: 10px 15px 15px; width: auto; height: 160px;' class="thumbnail" src="{{ MEDIA_URL }}{{ album.cover.file }}"></a>
```

Comment:
Must an album have a cover?  If it's optional, this may be trouble.  Also, consider using `album.cover.file.url` to get the media url already pre-rendered as part of the path expression instead of having to add it separately.